rhinoplasty/__init__.py

```diff
@@ -25,4 +25,4 @@
 from ._version import __version__
 
 # Version tuple
-version_info = tuple(map(int, re.match("(\d+)\.(\d+)\.(\d+)", __version__).groups()))
\ No newline at end of file
+version_info = tuple(int(elem) for elem in re.match("(\\d+)\\.(\\d+)\\.(\\d+)", __version__).groups())
```

Comment:
Yeah, the list comprehension is easier to read. I think a better way than double-slashes is to use the `r` string prefix, but this works too.

I'm surprised the old code worked, TBH (maybe it didn't...)